src/v/security/acl.h

```diff
@@ -97,29 +97,6 @@ enum class acl_operation : int8_t {
     idempotent_write = 10,
 };
 
-/*
- * Compute the implied operations based on the specified operation.
- */
-inline std::vector<acl_operation> acl_implied_ops(acl_operation operation) {
```

Comment:
i suppose you could return const-ref iterator pointers to static constexpr arrays as a `std::pair<>` but this is a good design. lgtm.